src/Stratis.SmartContracts.Executor.Reflection/Stratis.SmartContracts.Executor.Reflection.csproj

```diff
@@ -13,4 +13,8 @@
     <ProjectReference Include="..\Stratis.SmartContracts.Core\Stratis.SmartContracts.Core.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Think this isn't meant to be here? Got a little folder with an x next to it in VS